meta/runtime.yml

```diff
@@ -1679,6 +1679,13 @@ plugin_routing:
         warning_text: You are using an internal name to access the community.general.keycloak_role
           modules. This has never been supported or documented, and will stop working
           in community.general 9.0.0.
+    identity.keycloak.keycloak_role_composites:
+      redirect: community.general.keycloak_role_composites
+      deprecation:
+        removal_version: 9.0.0
+        warning_text: You are using an internal name to access the community.general.keycloak_role_composites
+          modules. This has never been supported or documented, and will stop working
+          in community.general 9.0.0.
```

Comment:
There should be no change to this file in this PR. This was only required before we removed the directory structure.

Reply:
I've removed this file as per origin/main

Reply:
If you removed that change locally you still have to push. The branch on GH still has this change.